bridge/bridge.go

```diff
@@ -9,6 +9,11 @@ type PdkBridge struct {
 	ch chan string
 }
 
+type Response struct {
+	Res *string
+	Err *string
```

Comment:
did you try without the pointers?  strings are already a reference type

Reply:
Done!